LBH.AdultSocialCare.Api/V1/Controllers/DayCarePackageController.cs

```diff
@@ -39,13 +52,47 @@ public async Task<IActionResult> CreateDayCarePackage([FromBody] DayCarePackageF
                     return UnprocessableEntity(ModelState);
```

Comment:
To effectively make use of the model state, the request model will need validation attributes applied, just one for the future, not required to change for this PR